rq_dashboard/web.py

```diff
@@ -340,5 +440,5 @@ def serialize_queue_names(worker):
 
 @blueprint.context_processor
 def inject_interval():
-    interval = current_app.config.get('RQ_DASHBOARD_POLL_INTERVAL')
+    interval = current_app.config.get('RQ_DASHBOARD_POLL_INTERVAL', 2500)
```

Comment:
Same here re: default values. The goal here is that I shouldn't have to grep for this all over the codebase to find multiple "default" values.

Reply:
It came from upstream.